docs/SRS/SRS.tex

```diff
@@ -113,9 +129,91 @@ \subsection{Undesired Scenario Handling}
 
 \section{System Level Variables}
 \subsection{Constants}
+
+\begin{table}[H]
+  \centering
+      \setlength{\leftmargini}{0.4cm}
+      \begin{tabular}{| >{\centering\arraybackslash}m{2cm} | 
+        >{\centering\arraybackslash}m{2cm} | 
+        >{\centering\arraybackslash}m{10cm} |}
+      \hline
+      \rowcolor[gray]{0.9}
+      Constant & Unit & Value\\
+      \hline
+      Chess board width & inches & 12\\
+     \hline
+     Chess board length & inches & 12\\
+     \hline
+     Chess board tile width & inches & 1.5\\
+     \hline 
+     Chess board tile length & inches & 1.5\\ 
+     \hline 
+      \end{tabular}
+  \label{Table}
+  \end{table}
```

Comment:
![image](https://user-images.githubusercontent.com/73441285/194167735-e458b367-3a6c-41b1-969f-4bd65c869737.png)
This table is being cut off from the page and I think the page number is getting merged in the table as well. Is there a way to fix that?